src/Tools/BuildValidator/Program.cs

```diff
@@ -278,102 +278,93 @@ private static bool ValidateFiles(IEnumerable<AssemblyInfo> assemblyInfos, Optio
             LocalSourceResolver sourceResolver,
             LocalReferenceResolver referenceResolver)
         {
-            MetadataReaderProvider? pdbReaderProvider = null;
-
-            try
+            // Find the embedded pdb
+            var originalBinaryStream = File.OpenRead(assemblyInfo.FilePath);
+            var originalPeReader = new PEReader(originalBinaryStream);
```

Comment:
Why remove the `using` from these two declarations?

Reply:
I believe I removed this because I was tracking down https://github.com/dotnet/roslyn/pull/52272#discussion_r605319192. At first I was concerned about reintroducing this because it looks like the PEReader can escape the method via CompilationDiff, but now I see that there is no field of type PEReader on CompilationDiff, so I think we're in the clear with that.

It feels like we might want to make a pass over data types which contain disposable things to determine if the containing type should be made disposable, and to check if things are actually getting disposed when they need to.